### PR TITLE
Remove mongoDisconnect () function

### PR DESCRIPTION
**Pull request by raconte**
Removed mongoDisconnect () function

### CODE REVIEW

File: test/unittests/mongoBackend/mongoQueryContext_test.cpp [hunk 1]
```diff
@@ -2993,7 +3043,7 @@ TEST(mongoQueryContextRequest, queryPattern0Attr)
     EXPECT_EQ("TA2", RES_CER_ATTR(0, 1)->type);
     EXPECT_EQ("val2", RES_CER_ATTR(0, 1)->value);
     EXPECT_EQ(SccOk, RES_CER_STATUS(0).code);
-    EXPECT_EQ("OK", RES_CER_STATUS(0).reasonPhrase);
+    EXPECT_EQ("OK", RES_CER_STATUS(0).reasonPhrase);
```
Comment (fgalan) [inline test/unittests/mongoBackend/mongoQueryContext_test.cpp]: Extra whitespace at the end of the line should be removed.

File: test/unittests/mongoBackend/mongoQueryContext_test.cpp [hunk 2]
```diff
@@ -503,8 +503,7 @@ TEST(mongoQueryContextRequest, paginationDetails)
     uriParams[URI_PARAM_PAGINATION_DETAILS]  = "on";
 
     /* Invoke the function in mongoBackend library */
-    long long count;
-    ms = mongoQueryContext(&req, &res, "", servicePathVector, uriParams, &count);
+    ms = mongoQueryContext(&req, &res, "", servicePathVector , uriParams);
```
Comment (fgalan) [inline test/unittests/mongoBackend/mongoQueryContext_test.cpp]: Pay attention to this...

**Comment by raconte** [comment]
LGTM

File: test/unittests/mongoBackend/mongoQueryContext_test.cpp (hunk 1 above)
Comment (fgalan) [inline test/unittests/mongoBackend/mongoQueryContext_test.cpp]: Fixed in f901c2d2e9f5d3221a656f0e37c0bfccd11fbe27

File: test/unittests/mongoBackend/mongoQueryContext_test.cpp (hunk 2 above)
Comment (fgalan) [inline test/unittests/mongoBackend/mongoQueryContext_test.cpp]: Fixed in 67f800a

**Comment by fgalan** [comment]
LGTM

**Pull request merged**